packages/s/shapelib/pspec_x86_64.xml

```diff
@@ -0,0 +1,73 @@
+        <Packager>
+            <Name>Muhammad Alfi Syahrin</Name>
+            <Email>malfisya.dev@hotmail.com</Email>
+        </Packager>
+        <License>LGPL-2.0-or-later</License>
+        <PartOf>programming.library</PartOf>
+        <Summary xml:lang="en">Simple C API for reading and writing ESRI Shapefiles</Summary>
+        <Description xml:lang="en">Simple C API for reading and writing ESRI Shapefiles
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>shapelib</Name>
+        <Summary xml:lang="en">Simple C API for reading and writing ESRI Shapefiles</Summary>
+        <Description xml:lang="en">Simple C API for reading and writing ESRI Shapefiles
+</Description>
+        <PartOf>programming.library</PartOf>
+        <Files>
+            <Path fileType="executable">/usr/bin/Shape_PointInPoly</Path>
+            <Path fileType="executable">/usr/bin/csv2shp</Path>
+            <Path fileType="executable">/usr/bin/dbfadd</Path>
+            <Path fileType="executable">/usr/bin/dbfcat</Path>
+            <Path fileType="executable">/usr/bin/dbfcreate</Path>
+            <Path fileType="executable">/usr/bin/dbfdump</Path>
+            <Path fileType="executable">/usr/bin/dbfinfo</Path>
+            <Path fileType="executable">/usr/bin/shpadd</Path>
+            <Path fileType="executable">/usr/bin/shpcat</Path>
+            <Path fileType="executable">/usr/bin/shpcentrd</Path>
+            <Path fileType="executable">/usr/bin/shpcreate</Path>
+            <Path fileType="executable">/usr/bin/shpdata</Path>
+            <Path fileType="executable">/usr/bin/shpdump</Path>
+            <Path fileType="executable">/usr/bin/shpdxf</Path>
+            <Path fileType="executable">/usr/bin/shpfix</Path>
+            <Path fileType="executable">/usr/bin/shpinfo</Path>
+            <Path fileType="executable">/usr/bin/shprewind</Path>
+            <Path fileType="executable">/usr/bin/shpsort</Path>
+            <Path fileType="executable">/usr/bin/shptreedump</Path>
+            <Path fileType="executable">/usr/bin/shputils</Path>
+            <Path fileType="executable">/usr/bin/shpwkb</Path>
+            <Path fileType="library">/usr/lib64/libshp.so.4</Path>
+            <Path fileType="library">/usr/lib64/libshp.so.4.0.0</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>shapelib-devel</Name>
+        <Summary xml:lang="en">Development files for shapelib</Summary>
+        <Description xml:lang="en">Simple C API for reading and writing ESRI Shapefiles
+</Description>
+        <PartOf>programming.devel</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="1">shapelib</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="header">/usr/include/shapefil.h</Path>
+            <Path fileType="library">/usr/lib64/libshp.a</Path>
```

Comment:
Don't need the static libs

I would say we can ditch the `-devel` package and just pattern everything into `shapelib`